frontend/javascripts/dashboard/dashboard_view.tsx

```diff
@@ -281,20 +289,26 @@ class DashboardView extends PureComponent<PropsWithRouter, State> {
       !activeUser.novelUserExperienceInfos.hasSeenDashboardWelcomeBanner ? (
         <WhatsNextHeader activeUser={activeUser} onDismiss={this.onDismissWelcomeBanner} />
       ) : null;
+    this.state.pricingPlanStatus?.isAlmostExceeded;
 
+    // ToDo enable components below once pricing goes live
```

Comment:
is there a (follow-up) issue which could be linked here in the comment?

Reply:
Nope.